components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -536,13 +539,13 @@ public int getBusinessOwnersCount(int tenantId) throws AppManagementException {
      * @return
      * @throws AppManagementException
      */
-    public List<Integer> getBusinessOwnerIdsBySearchPrefix(String searchPrefix, int tenantId) throws AppManagementException {
```

Comment:
Java doc comments and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files